### PR TITLE
More flinkcluster->flink renaming.

### PR DESCRIPTION
**Pull request by analogue**
This should have been part of https://github.com/Yelp/paasta/pull/2215

### CODE REVIEW

**Comment by solarkennedy** [comment]
@Rob-Johnson can you review this, is it still relevant?

File: tests/test_setup_kubernetes_cr.py [hunk 1]
```diff
@@ -66,15 +66,15 @@ def test_setup_all_custom_resources():
 
         mock_client = mock.Mock()
         flink_crd = mock.Mock()
-        flink_crd.spec.names = mock.Mock(plural='flinkclusters', kind='FlinkCluster')
+        flink_crd.spec.names = mock.Mock(plural='flink', kind='Flink')
```
Comment (Rob-Johnson) [inline tests/test_setup_kubernetes_cr.py]: this should be flinks

File: tests/test_setup_kubernetes_cr.py [hunk 2]
```diff
@@ -66,15 +66,15 @@ def test_setup_all_custom_resources():
...
         cassandra_crd = mock.Mock()
         cassandra_crd.spec.names = mock.Mock(plural='cassandraclusters', kind='CassandraCluster')
         mock_client.apiextensions.list_custom_resource_definition.return_value = mock.Mock(
             items=[flink_crd, cassandra_crd],
         )
 
         custom_resources = [
-            mock.Mock(kube_kind=mock.Mock(plural='flinkclusters', singular='FlinkCluster')),
+            mock.Mock(kube_kind=mock.Mock(plural='flink', singular='Flink')),
```
Comment (Rob-Johnson) [inline tests/test_setup_kubernetes_cr.py]: same

**Comment by Rob-Johnson** [comment]
@analogue I'm closing this - reopen if you want to come back and finish what you started

**Pull request closed without merging**